sys-apps/plocate/files/plocate-1.1.16-include-linux_stat_h-musl.patch

```diff
@@ -0,0 +1,25 @@
+Similar patch is already upstreamed https://git.sesse.net/?p=plocate;a=commit;h=0125004cd28c5f9124632b594e51dde73af1691c
```

Comment:
Why don't we take the upstream patch verbatim?

Reply:
I noticed there was a commit after I patched it so I just kept mine. I sent a better patch upstream with fprintf + exit so I guess I'll just wait him to get back on that.

Reply:
This should work: https://git.sesse.net/?p=plocate;a=patch;h=0125004cd28c5f9124632b594e51dde73af1691c

Reply:
Updated @ArsenArsen

Reply:
Updated again @ArsenArsen :). Got my patch upstream so this should be okay to merge.

Reply:
seems good